lib/src/components/NxAccordion/__tests__/NxAccordion.test.tsx

```diff
@@ -5,313 +5,202 @@
 
 describe('NxAccordion', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxAccordion, {}),
-      getMountedComponent = enzymeUtils.getMountedComponent<Props>(NxAccordion, {});
+  const quickRender = rtlRender(NxAccordion, {});
+  const renderEl = rtlRenderElement(NxAccordion, {});
 
   it('renders a <details> element with the provided props', function() {
-    const component = getShallowComponent({ id: 'foo', open: true, title: 'test title' });
-
-    expect(component).toMatchSelector('details#foo');
-    expect(component).toHaveProp('open', true);
-    expect(component).toHaveProp('title', 'test title');
+    const el = renderEl({ id: 'foo', open: true })!;
+    expect(el.tagName).toBe('DETAILS');
+    expect(el).toHaveAttribute('id', 'foo');
+    expect(el).toHaveAttribute('open');
   });
 
-  it('sets the nx-accordion class along with any provided className', function() {
-    expect(getShallowComponent()).toHaveClassName('nx-accordion');
-    expect(getShallowComponent({ className: 'foo' })).toHaveClassName('nx-accordion');
-    expect(getShallowComponent({ className: 'foo' })).toHaveClassName('foo');
+  it('sets the provided className', function() {
+    const el = renderEl()!;
+    const customEl = renderEl({ className: 'foo' })!;
+
+    expect(customEl).toHaveClass('foo');
+
+    for (const cls of Array.from(el.classList)) {
+      expect(customEl).toHaveClass(cls);
+    }
   });
 
   it('sets an id if none is specified', function() {
-    expect(getShallowComponent().prop('id')).toBeTruthy();
+    expect(renderEl()).toHaveAttribute('id');
   });
 
   it('sets aria-expanded from the open prop', function() {
-    expect(getShallowComponent()).toHaveProp('aria-expanded', false);
-    expect(getShallowComponent({ open: false })).toHaveProp('aria-expanded', false);
-    expect(getShallowComponent({ open: true })).toHaveProp('aria-expanded', true);
+    expect(renderEl()).toHaveAttribute('aria-expanded', 'false');
+    expect(renderEl({ open: false })).toHaveAttribute('aria-expanded', 'false');
+    expect(renderEl({ open: true })).toHaveAttribute('aria-expanded', 'true');
   });
 
-  it('renders non-header children in an nx-accordion__content wrapper', function() {
-    const component =
-        mount(
-          <NxAccordion>
-            <NxAccordion.Header>
-              <span>Foo</span>
-            </NxAccordion.Header>
-            <span>Bar</span>
-          </NxAccordion>
-        ),
-        childrenWrapper = component.find('.nx-accordion__content');
-
-    expect(childrenWrapper).toExist();
-    expect(childrenWrapper).toContainReact(<span>Bar</span>);
-    expect(childrenWrapper).not.toContainReact(<span>Foo</span>);
+  it('renders non-header children in content wrapper', function() {
+    const { container } = quickRender({ children: (
+      <NxAccordion.Header>
+        <span>Foo</span>
+      </NxAccordion.Header>
+
+    ),
+    className: 'bar' });
```

Comment:
Something got messed up here.  The `bar` class needs to be applied to non-header children, not to the accordion itself

Reply:
> Something got messed up here. The `bar` class needs to be applied to non-header children, not to the accordion itself

Fixed here: 323786008e9a71801d1165fd650fa9b0c4cc49b5

Reply:
> Fixed here: https://github.com/sonatype/sonatype-react-shared-components/commit/323786008e9a71801d1165fd650fa9b0c4cc49b5

Why'd you switch it away from `quickRender`?

Reply:
> > Fixed here: [3237860](https://github.com/sonatype/sonatype-react-shared-components/commit/323786008e9a71801d1165fd650fa9b0c4cc49b5)
> 
> Why'd you switch it away from `quickRender`?

Because when I quickRender and then specify an Accordion within it would that be rendering an accordion within an accordion similar to what I was doing with the headers before?

Reply:
> Because when I quickRender and then specify an Accordion within it would that be rendering an accordion within an accordion similar to what I was doing with the headers before?

So don't add an extra layer of accordions.  Do it like you had at the beginning of this thread, but remove the `className: 'bar'` from the `quickRender` properties and add an element with `className="bar"` as a sibling to the `NxAccordion.Header`.

Oh, you'll need a fragment to wrap the header and sibling.  Is that what was tripping you up?

Reply:
> > Because when I quickRender and then specify an Accordion within it would that be rendering an accordion within an accordion similar to what I was doing with the headers before?
> 
> So don't add an extra layer of accordions. Do it like you had at the beginning of this thread, but remove the `className: 'bar'` from the `quickRender` properties and add an element with `className="bar"` as a sibling to the `NxAccordion.Header`.
> 
> Oh, you'll need a fragment to wrap the header and sibling. Is that what was tripping you up?

Yeah when I do this: 
`const { container } = quickRender(
      <>
        <NxAccordion.Header>
          <span>Foo</span>
        </NxAccordion.Header>
        <span className='bar'>Bar</span>
      </>
    );`

I get that it has a value of null

Reply:
> Yeah when I do this:
const { container } = quickRender( <> <NxAccordion.Header> <span>Foo</span> </NxAccordion.Header> <span className='bar'>Bar</span> </> );
>
> I get that it has a value of null

Don't forget that the parameter to `quickRender` should be the entire props object, not just the children.  I have pushed a commit that addresses this.